sam_dead_reckoning/launch/dual_ekf_test.launch

```diff
@@ -362,6 +362,13 @@
         <remap from="set_pose" to="/initialpose"/>
     </node>
 
+        <!--Including tf_convenience topics-->
+    	<node name="odom_listener" pkg="tf_convenience_topics" type="odom_listener" output="screen" >
+	    	<param name="odom_topic" value="/$(arg robot_name)/dr/odom"/>
+	    	<param name="topic_from_dvl" value="$(arg robot_name)/core/dvl"/>
```

Comment:
@svbhat this is gonna be an issue. You need to have a `/` at the start, otherwise it's gonna be namespaced.

Reply:
Updated that now, I've also updated the odom topic to /dr/odom by default.

Reply:
Perfect